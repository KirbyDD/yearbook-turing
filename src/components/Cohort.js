import React from 'react';
import Person from './Person';
import AddNewStudent from './AddNewStudent'
import './Cohort.css';

const Cohort = ({staff, students, addStudent}) => {
  const staffList = staff.map(member => {
    return <Person name={member.name} photo={member.photo} 
              quote={member.quote} superlative={member.superlative}/>
  })
  const studentList = students.map(member => {
    return <Person name={member.name} photo={member.photo} 
              quote={member.quote} superlative={member.superlative}/>
  })
  return (
    <main>
      <h2>Staff</h2>
      <div>
        {staffList}
      </div>
      <h2>Students</h2>
      <div>
        {studentList}
      </div>
      <AddNewStudent addStudent={addStudent}/>>
    </main>
  )
}

export default Cohort;
