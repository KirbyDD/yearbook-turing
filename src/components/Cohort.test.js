import React from 'react';
import Cohort from './Cohort';
import { shallow } from 'enzyme'

describe('Cohort', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Cohort staff={[]} students={[]}/>)
    expect(wrapper).toMatchSnapshot();
  })
})