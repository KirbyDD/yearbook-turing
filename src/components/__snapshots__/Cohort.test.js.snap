// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cohort should match snapshot 1`] = `
<main>
  <h2>
    Staff
  </h2>
  <div />
  <h2>
    Students
  </h2>
  <div />
  <AddNewStudent />
  &gt;
</main>
`;
